### ISSUE TITLE
Bell does not ring when meditation session ends 

### ISSUE THREAD

**Issue by stliang** [Sirimangalo/meditation-plus-angular#300]
I thought the point of the bell is notify user when meditation session ends.  If the is not the case, would be great to have this feature.

**Comment by olastor** [comment]
Yeah, sorry about that. The problem is well known (#209). It would be good to know which platform (smartphone, Windows, Mac, Linux ...) and Browser (Firefox, Chrome, Safari) you're using.

**Comment by bkeevil** [comment]
Doesn't work on firefox on Android

**Comment by olastor** [comment]
Thanks for the hint, couldn't reproduce the error myself. Will check it with firefox
